apple/RCTMarkdownUtils.mm

```diff
@@ -42,24 +27,21 @@ - (NSAttributedString *)parseMarkdown:(nullable NSAttributedString *)input withA
         }
 
         jsi::Runtime &rt = *runtime;
-        auto text = jsi::String::createFromUtf8(rt, [inputString UTF8String]);
+        auto text = jsi::String::createFromUtf8(rt, [attributedString.string UTF8String]);
```

Comment:
I'm not sure about the look of mixing styles here. I think we should go with either `[[attributedString string] UTF8String]` or `attributedString.string.UTF8String`

Reply:
Fixed in 5a1651f